tools/http3_test/src/runner.rs

```diff
@@ -28,7 +28,7 @@ use std::net::ToSocketAddrs;
 
 use ring::rand::*;
 
-pub fn run(test: &mut crate::Http3Test, verify_peer: bool) {
+pub fn run(test: &mut crate::Http3Test, host: &str, verify_peer: bool) {
```

Comment:
Instead of a string, maybe pass a `SocketAddr` directly? https://doc.rust-lang.org/std/net/enum.SocketAddr.html

Reply:
I did that first but then realised it's a bit annoying if you just want to pass a string directly into runner.rs. 

It's a pretty close call on which is better.

Reply:
It was actually easier to handle some edge cases (related to your other comment) by only accepting a SocketAddr